docs/technical_reference/unit_models/osmotically_assisted_reverse_osmosis_0D.rst

```diff
@@ -0,0 +1,198 @@
+Osmotically Assisted Reverse Osmosis (0D)
+=========================================
+This osmotically assisted reverse osmosis (OARO) unit model
+   * is 0-dimensional
+   * supports a single liquid phase only
+   * supports steady-state only
+   * is based on the solution-diffusion model and film theory
+   * assumes isothermal conditions
+   * assumes the feed-side flows in the forward direction
+   * assumes the permeate-side flows in the backwards direction
+
+.. index::
+   pair: watertap.unit_models.osmotically_assisted_reverse_osmosis_0D;osomtically_assisted_reverse_osmosis_0D
+
+.. currentmodule:: watertap.unit_models.osmotically_assisted_reverse_osmosis_0D
+
+Degrees of Freedom
+------------------
+Aside from the inlet feed state variables (i.e. temperature, pressure, component flowrates), the OARO model has
+at least 5 degrees of freedom that should be fixed for the unit to be fully specified. Unlike RO, which only
+accounts for concentration polarization on the feed side, the OARO model includes a structural parameter
+variable, which is used to calculate the membrane-interface concentration on the permeate side.
+
+
+Typically, the following variables are fixed for the OARO model, in addition to state variables at the inlet:
+    * membrane water permeability, A
+    * membrane salt permeability, B
+    * permeate pressure
+    * membrane area
+    * structural parameter
+
+On the other hand, configuring the OARO unit to calculate concentration polarization effects, mass transfer
+coefficient, and pressure drop would result in 7 additional degrees of freedom. In this case, in addition to the
+previously fixed variables, we typically fix the following variables to fully specify the unit:
+
+    * feed-spacer porosity
+    * feed-channel height
+    * feed-velocity
+    * permeate-space porosity
+    * permeate-channel height
+    * permeate-side velocity
```

Comment:
@adam-a-a Do we need to specify `permeate-side velocity`? I checked the test file, it seems only `feed-side velocity` is fixed.